src/pages/affiliates/AffiliatesPage.tsx

```diff
@@ -0,0 +1,155 @@
+import { Suspense } from 'react';
+
+import { Outlet, useLocation, useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { IProgramStats } from '@/constants/affiliates';
+import { STRING_KEYS } from '@/constants/localization';
+import { AffiliateRoute } from '@/constants/routes';
+
+import { useAccounts } from '@/hooks/useAccounts';
+import { useAffiliatesInfo } from '@/hooks/useAffiliatesInfo';
+import { useBreakpoints } from '@/hooks/useBreakpoints';
+import { useStringGetter } from '@/hooks/useStringGetter';
+import { useWalletConnection } from '@/hooks/useWalletConnection';
+
+import { layoutMixins } from '@/styles/layoutMixins';
+
+import { AttachedExpandingSection } from '@/components/ContentSection';
+import { LoadingSpace } from '@/components/Loading/LoadingSpinner';
+import { NavigationMenu } from '@/components/NavigationMenu';
+import { AffiliatesLeaderboard } from '@/views/Affiliates/AffiliatesLeaderboard';
+import LastUpdated from '@/views/Affiliates/LastUpdated';
+import { ShareAffiliateBanner } from '@/views/Affiliates/ShareAffiliateBanner';
+import { AffiliateStatsCard } from '@/views/Affiliates/cards/AffiliateStatsCard';
+import { ProgramStatsCard } from '@/views/Affiliates/cards/ProgramStatsCard';
+import { ProgramStatusCard } from '@/views/Affiliates/cards/ProgramStatusCard';
+import { AffiliatesBanner } from '@/views/AffiliatesBanner';
+
+export const AffiliatesPage = () => {
+  const navigate = useNavigate();
+  const location = useLocation();
+
+  const { isConnectedWagmi } = useWalletConnection();
+  const { dydxAddress } = useAccounts();
+  const { programStatsQuery, affiliateStatsQuery, lastUpdatedQuery, affiliateMetadataQuery } =
+    useAffiliatesInfo(dydxAddress);
+  const { data: lastUpdated } = lastUpdatedQuery;
+  const { data: accountStats } = affiliateStatsQuery;
+  const { data: programStats } = programStatsQuery;
+  const { data: affiliateMetadata } = affiliateMetadataQuery;
```

Comment:
[q][non-blocking] is returning the full query object from a hook recommended? if we're only using the `data` property feels like it might be slightly easier to just export the data itself.